app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">MCStatus</string>
+    <string name="about_version">Version %1$s (Build %2$d)</string>
+    <string name="action_refresh">Neuladen</string>
+    <string name="action_addserver">Server hinzufügen</string>
+    <string name="action_deleteserver">Server(s) löschen</string>
+    <string name="action_editserver">Server bearbeiten</string>
+    <string name="action_about">Über diese App</string>
+    <string name="action_help">Hilfe</string>
+    <string name="action_privacy">Privacy</string>
+    <string name="icon_description">Server Bild</string>
+    <string name="edit_server_name">Server Namen eingeben:</string>
+    <string name="edit_server_name_hint">Mein Minecraft Server</string>
+    <string name="edit_server_address">Server Addresse eingeben:</string>
```

Comment:
The spellcheck in Android Studio is complaining that this should be 'Adresse', is that correct?

Reply:
Yes, my bad 'Adresse' is right